src/Compilers/VisualBasic/Portable/Symbols/SubstitutedNamedType.vb

```diff
@@ -589,6 +590,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Next
         End Function
 
+        Friend Overrides ReadOnly Property AllRequiredMembers As ImmutableSegmentedDictionary(Of String, Symbol)
+            Get
+                Dim originalRequiredMembers = OriginalDefinition.AllRequiredMembers
+                If originalRequiredMembers.IsEmpty OrElse originalRequiredMembers = s_requiredMembersErrorSentinel Then
+                    Return originalRequiredMembers
+                End If
+
+                Dim members = GetMembersUnordered()
+                Dim builder = originalRequiredMembers.ToBuilder()
+                For Each kvp In originalRequiredMembers
+                    builder(kvp.Key) = SubstituteTypeParametersInMember(kvp.Value)
```

Comment:
>SubstituteTypeParametersInMember(kvp.Value) [](http://example.com/codeflow?start=39&length=43)

Are we testing effect of this fix?

Reply:
`GenericSubstitution_NoneSet` and `GenericSubstitution_AllSet`.